.release-please-manifest.json

```diff
@@ -1,11 +1,11 @@
 {
-    "solutions/client-landing-zone": "0.0.1",
-    "solutions/client-project-setup": "0.0.1",
-    "solutions/client-setup": "0.0.1",
-    "solutions/core-landing-zone": "0.0.1",
-    "solutions/experimentation/admin-folder": "0.0.1",
+    "solutions/client-landing-zone": "0.1.0",
+    "solutions/client-project-setup": "0.1.0",
+    "solutions/client-setup": "0.1.0",
+    "solutions/core-landing-zone": "0.1.0",
+    "solutions/experimentation/admin-folder": "0.1.0",
     "solutions/experimentation/client-landing-zone": "0.0.1",
```

Comment:
this one and experimentation/core-landing-zone is missing...investigating

Reply:
PR #386 to re-trigger on missing packages

Reply:
good to go now

Reply:
Nice  work Alain!